packages/lodestar/src/chain/blocks/importBlock.ts

```diff
@@ -187,6 +192,35 @@ export async function importBlock(chain: ImportBlockModules, fullyVerifiedBlock:
     const cp = getCheckpointFromState(checkpointState);
     chain.checkpointStateCache.add(cp, checkpointState);
     pendingEvents.push(ChainEvent.checkpoint, cp, checkpointState);
+
+    // Note: in-lined code from previos handler of ChainEvent.checkpoint
+    chain.logger.verbose("Checkpoint processed", toCheckpointHex(cp));
```

Comment:
Unwanted diff

Reply:
This must be a result of the recent issues we have with our git merging strategies. If you look at the commits [here](https://github.com/ChainSafe/lodestar/pull/4142/commits) you see commits to this branch by you and @wemeetagain I don't think that is normal

@wemeetagain what do you suggest?